lib/ansible/modules/cloud/amazon/ecs_service.py

```diff
@@ -180,6 +212,55 @@
 '''
 import time
 
+DEPLOYMENT_CONFIGURATION_TYPE_MAP = {
+    'maximum_percent': 'int',
+    'minimum_healthy_percent': 'int'
+}
+
+class TypeMapper:
```

Comment:
these would belong in the shared code in module_utils, not a show stopper at this point but consider moving them there as we already have 'decamelize' function also